sourcegraph/about
# Starship branding on 4.0 page

Updates the Sourcegraph 4.0 page with Starship branding; updated hero ~and cta~

### Changelog
- Closes #5702 by updating the hero with starship branding and a few rearrangements
- Updated starship background gradients
- Added our new starship background svg for use with our `Background` component
- Adds a different variant for the starship logo which has less blur on the reflection
- Fixes a bug where the footer did not have a black background for the `navbar-dark` class
- Modified `ContentSection` and `CtaSection` components to inherit our Background component backgrounds
- Changed the API `button` prop on `CtaSection` to `ctaStyle` and updated all references
- Changed the API `color` prop on `ContentSection` to `background` and updated all references
- Changed the API `backButton` prop on `Hero` from `ReactNode` to an object with `link` and `text` to scope and standardize the component composition
- ~Added `CtaSection` to the 4.0 page with the starship branding~
- Cleaned up some components with more code reuse and removed our old `TrySourcegraph` component in lieu of our new standardized `CtaSection`
- Updated and replaced our dark nebulous gradient to use a code gradient and removed the image gradient background from our Background component

### Test

1. Ensure prettier has standardized the proposed changes.
2. Ensure page is neatly responsive

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202885951329736
